### DISCUSSION TITLE
Why does the Popover have a minimum of left: 12px?

### DISCUSSION THREAD

**Discussion by heisand** [adobe/react-spectrum#5009] [Q&A] [answered]
I am just wondering why the `Popover` from `usePopover` has a minimum of `left: 12px`? It at least seems that this is the minimum. When there is less than 12 px to the left of the Popover to the parent container, the list box in a ComboBox will be displaced relative to the input field (as you can see from screenshot 2 where I have modified the [example](https://codesandbox.io/s/hardcore-moon-xzc4r?file=/src/App.tsx) and removed all the padding to the left).

Could there be some way to not use a minimum left value, and instead let the value decrease towards 0 with decreasing padding to the left (at least in combination with a `useComboBox` (or make it configurable))?

**1.** 
<img width="896" alt="Skjermbilde 2023-08-31 kl  21 41 42" src="https://github.com/adobe/react-spectrum/assets/15145686/52fc6127-d380-486d-bf51-cfc557811d05">

**2.**
![Skjermbilde 2023-08-31 kl  17 16 14](https://github.com/adobe/react-spectrum/assets/15145686/275926b6-9083-4155-9694-5e814d6215db)

**Comment by snowystinger** [comment]
It's used by our Spectrum implementation, but that seems too opinionated for our hooks/RAC implementations. I think we could consider changing that.

**Reply by heisand** [reply]
It would be really great if that could be changed!

**Comment by reidbarber** [comment, marked as answer]
You can pass `containerPadding: 0` to `usePopover` to override this. The RAC Popover also supports this prop.

https://react-spectrum.adobe.com/react-aria/usePopover.html#container-padding

https://react-spectrum.adobe.com/react-aria/Popover.html#container-padding

The default is `12px`, but I'm not sure we want to change the default now since people may be relying on the default behavior.

**Reply by snowystinger** [reply]
o, good memory, I forgot about that. Yeah, we definitely wouldn't want to change the default we have.

**Reply by heisand** [reply]
Oh, thanks!
